### ISSUE TITLE
Issue with metadata retrieval

### ISSUE THREAD

**Issue by kalvn** [shaarli/Shaarli#1756]
I found this link for which the metadata retrieval seems broken :

https://material.io/blog/announcing-material-you

![Screenshot_20210519-131234~2](https://user-images.githubusercontent.com/6149640/118805145-e241a680-b8a5-11eb-9fa0-51425c092e44.jpg)

**Comment by nodiscc** [comment]
@kalvn is this bug still present? I cannot reproduce it on my own instance (v0.12.2) and on https://demo.shaarli.org/admin/add-shaare -> `Add link` -> description is correctly extracted

**Comment by kalvn** [comment]
Yes I still have it, but I'm still running 0.12.1 so it might have been fixed in later versions. I'll try to find some time to run some tests on a fresh instance.

**Comment by nodiscc** [comment]
@kalvn hi, any news? Did you get a chance to recheck on v0.12.2?

**Comment by kalvn** [comment]
Yes and it seems better. I still have a case from time to time where it fails but I suspect more something related to websites themselves. I close the issue.

**Comment by virtadpt** [comment]
I run into this more often than not, and it seems that it's because the metadata is just not available at the page linked to, or it's messed up somehow.  This isn't a Shaarli thing.
